.github/workflows/on_release_published__build_publish_release.yml

```diff
@@ -1,16 +1,8 @@
-name: on_push_tag_build_publish_release
+name: on_release_published__build_publish_release
 
 on:
-  workflow_dispatch:
-    docker_tag:
-      description: "Define the docker tag name. You can define the docker tag version separately"
-      default: mindwendel
-    docker_tag_version:
-      description: "Define the docker tag version"
-      required: true
-  push:
-    tags:
-      - "*.*.*"
+  release:
```

Comment:
I decided to switch to the release event as this is a better trigger to initiate a release